### ISSUE TITLE
requests > 0.14.2 breaks rawes

### ISSUE THREAD

**Issue by dylanvaughn** [humangeo/rawes#9]
Hi!  When I try to initialize rawes with requests 1.0.3 I get something like this:

```
     SEARCH = rawes.Elastic('127.0.0.1:9200')
  File "/home/dylan/projects/myproj/local/lib/python2.7/site-packages/rawes-0.3.2-py2.7.egg/rawes/elastic.py", line 47, in __init__
    self.connection = HttpConnection(self.host, self.port, timeout=self.timeout)
  File "/home/dylan/projects/myproj/local/lib/python2.7/site-packages/rawes-0.3.2-py2.7.egg/rawes/http_connection.py", line 31, in __init__
    self.session = requests.session(timeout=timeout)
  File "/home/dylan/projects/myproj/local/lib/python2.7/site-packages/requests-1.0.3-py2.7.egg/requests/sessions.py", line 385, in session
    return Session(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'timeout'
```

Looks like 'timeout' was removed as an option here:

https://github.com/kennethreitz/requests/commit/92355ada54a3a19341f7fbc65a8bc50816858c63#L0L159

Maybe change this: e3d4956fe6e5948b187d2675954f0f08e294f4e8 to go back to 0.14.2?

**Comment by dwnoble** [comment]
Damn.

I suppose that is the danger of an unbounded >= in your requirements file.

I've changed the requirement to 'requests>=0.11, <1.0.0'

A few people were having an issue with 'requests==0.14.2' where another library was requiring specifically 'requests==0.1x.x' (not precicely 0.14.2), and thus conflicting with rawes.  Hence the original motivation behind the change.  Let me know if this new commit works for you.

**Comment by angelo-romano** [comment]
Hi, it would be nice to have improved support for more recent versions of requests (I am using a more recent version to a project of mine, namely).

**Comment by dwnoble** [comment]
Hi Angelo,

I looked into this issue and it turned out to be a pretty easy fix so i pushed up version 0.3.6 that will work with the most recent version of requests.  Let me know if this works for you!
